api_examples/example.py

```diff
@@ -21,6 +22,9 @@ def main():
     data = r.json()
     # data = {"zipFile": "...", "process": "...", "format": "..."}
 
+    # download with a single request (PING-PONG)
+    # open(data["zipFile"] + ".xlsx", 'wb').write(base64.b64decode(data["process"]))
```

Comment:
the name "zipFile" might be misleading after the fix, this line will write an xlsx file directly and not a zip file.